src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/Time.java

```diff
@@ -0,0 +1,49 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects;
+
+public class Time {
+   private int hours;
+   private int minutes;
+   private int seconds;
+
+    Time() {
+    }
+
+    Time(int hours, int minutes, int seconds) {
+        this.hours = hours;
+        this.minutes = minutes;
+        this.seconds = seconds;
+    }
+
+    Time (int totalSeconds) {
+        this(0, 0, totalSeconds);
+    }
```

Comment:
Why does specification contain 2 variables for one business value?
`seconds` != `totalSeconds`